data.json

```diff
@@ -1,55 +1,50 @@
 {
```

Comment:
Since this is the file type that we're using the the transaction log portion, would it be better if it was named something else, so that for actually using the server, not for testing, we'll get accurate values?